submissions/Oodmincheg/js-tiny-world/index.js

```diff
@@ -0,0 +1,71 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const Inhabition = function(specie, name, gender, legs, hands, saying) {
+  this.specie = specie;
+  this.name = name;
+  this.gender = gender;
+  this.legs = legs;
+  this.saying = saying;
+  this.handsString = hands === 0 ? "no" : this.hands;
```

Comment:
```suggestion
  this.handsString = hands === 0 ? "no" : hands;
```
There is no `hands` property (`this.hands`) in this class.